pages/docs/open-source.mdx

```diff
@@ -15,9 +15,15 @@ Learn more about what's next for Langfuse on our [roadmap](/docs/roadmap).
 
 ## License
 
-Langfuse is licensed under an [MIT license](https://github.com/langfuse/langfuse/blob/main/LICENSE), with the exception of the `/ee` folder of the repository, if applicable.
+Langfuse is licensed under an MIT license, with the exception of the `/ee` and `/web/src/ee` folders of the repository. These contain features that are available within the Enterprise Edition of Langfuse and on Langfuse Cloud. See [License](https://github.com/langfuse/langfuse/blob/main/LICENSE) for more details.
 
-Currently, our focus is entirely on open source. However, we may consider commercializing premium features as part of an enterprise version in the future. For specific requirements, please contact us at enterprise@langfuse.com
+All Langfuse-maintained integrations and SDKs are fully open-source and licensed under the MIT license.
```

Comment:
The Langfuse core product and all Langfuse-maintained integrations and SDKs are fully open-source and licensed under the MIT license.